src/javascript/_common/third_party_links.js

```diff
@@ -0,0 +1,50 @@
+const State            = require('./storage').State;
+const Client           = require('../app/base/client');
+const BinarySocket     = require('../app/base/socket');
+const Dialog           = require('../app/common/attach_dom/dialog');
+
+const ThirdPartyLinks = (() => {
+    const init = () => {
+        if (Client.isLoggedIn()) {
+            BinarySocket.wait('landing_company').then(() => {
+                if (State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest') {
```

Comment:
instead of getting it from state you could get response from `then`, e.g. `then((response) => { if (getPropertyValue(response, ['landing_company', 'financial_company', 'shortcode']) ... `